opensubtitles/infos.go

```diff
@@ -0,0 +1,96 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+// InfoService provides access to the info related functions
+// in the OpenSubtitles API.
+//
+// OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#infos
+type InfoService service
+
+//UserData represent all the information of a User
+type UserData struct {
+	Jti                string `json:"jti"`
+	AllowedDownloads   int    `json:"allowed_downloads"`
+	Level              string `json:"level"`
+	UserID             int    `json:"user_id"`
+	ExtInstalled       bool   `json:"ext_installed"`
+	Vip                bool   `json:"vip"`
+	RemainingDownloads int    `json:"remaining_downloads"`
+}
+
+//Languages contains all the LanguagesData
+type Languages struct {
+	Data []LanguagesData `json:"data"`
+}
+
+//LanguagesData associates a code to a name
+type LanguagesData struct {
+	LanguageCode string `json:"language_code"`
+	LanguageName string `json:"language_name"`
+}
+
+//Languages get the languages used on opensubtitles and their codes
+//OpenSubtitles API docs : https://www.opensubtitles.com/docs/api/html/index.htm#get-the-languages-table
+func (s *InfoService) Languages(ctx context.Context) (*Languages, *http.Response, error) {
+	u := "/api/v1/infos/languages"
+	req, err := s.client.NewRequest("GET", u, "")
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var languages *Languages
+	resp, err := s.client.Do(ctx, req, &languages)
+	if err != nil {
+		return nil, resp, err
+	}
+	return languages, resp, nil
+}
+
+type Formats struct {
```

Comment:
Codacy found an issue: [exported type Formats should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)